### DISCUSSION TITLE
노트북 내장 스피커, 마이크 사용 시 하울링이 생깁니다.

### DISCUSSION THREAD

**Discussion by DolDolDol** [metamong-friends/metamong-friends.github.io#13] [제보하기]
한 명이 이어폰을 끼고 이야기를 하고, 다른 한명은 노트북 장비에 탑재된 스피커와 마이크를 켜놓고 있는 상태에서.
이어폰 낀 사람이 이야기 하면 그 사람이 이야기 한게 하울링이 생기네요. 
여러 명이 그렇게 있으면 알아 듣기 힘들 정도여서, 일단 말 할때만 마이크 켜고 이야기 하는 걸로 하고 있습니다.

**Comment by jihwanKo** [comment]
요거 1순위로 생각해주시면 좋을거 같아요!

**Comment by jihwanKo** [comment]
가능하다면 지연시간도.. 줄여주세요~ ㅎㅎ

**Comment by ACT-TJ** [comment]
네 지연시간은 좀 쉽게 조정이 가능할 듯 하여 바로 반영해 보도록 할게요. 
또 피드백 주시면 너무 감사할 것 같습니다.

하울링 및 음성 세부조정은 조금 시간을 갖고 조정을 해볼텐데, 변화가 생겨
적용하게 되면 릴리즈 안내를 드리도록 하겠습니다. 

의견 너무 감사합니다!

**Comment by shlee303** [comment]
하울링 조정 반영됐습니다 :)! 좋은 피드백 주셔서 감사합니다~
